packages/mobile/src/screens/search-screen/SearchBar.tsx

```diff
@@ -1,56 +1,55 @@
 import { useCallback, useRef, useState } from 'react'
 
+import { fetchSearch } from 'audius-client/src/common/store/search-bar/actions'
+import { getSearchBarText } from 'audius-client/src/common/store/search-bar/selectors'
+import debounce from 'lodash/debounce'
 import { useDispatch, useSelector } from 'react-redux'
 
 import type { TextInputRef } from 'app/components/core'
 import { TextInput } from 'app/components/core'
 import LoadingSpinner from 'app/components/loading-spinner'
-import { useDispatchWeb } from 'app/hooks/useDispatchWeb'
 import { useNavigation } from 'app/hooks/useNavigation'
-import { MessageType } from 'app/message'
 import { updateQuery } from 'app/store/search/actions'
-import {
-  getSearchQuery,
-  getSearchResultQuery
-} from 'app/store/search/selectors'
-import { getTagSearchRoute } from 'app/utils/routes'
+import { getSearchQuery } from 'app/store/search/selectors'
 
 export const SearchBar = () => {
   const query = useSelector(getSearchQuery)
   const dispatch = useDispatch()
-  const dispatchWeb = useDispatchWeb()
   const navigation = useNavigation()
   const [clearable, setClearable] = useState(query !== '')
   const inputRef = useRef<TextInputRef>(null)
 
-  const handleChangeText = useCallback(
-    (text: string) => {
-      dispatch(updateQuery(text))
+  // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Which dep is it complaining about?

Reply:
it was complaining because it can't calculate the deps `React Hook useCallback received a function whose dependencies are unknown.` - added a comment to explain